<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="portLabel">
      <property name="text">
       <string>端口号：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="portLE"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>621</width>
     <height>47</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>客户端消息：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="clientInfoLE">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>470</y>
     <width>541</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="startPB">
      <property name="text">
       <string>启动</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closePB">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>621</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="clientSocketLabel">
      <property name="text">
       <string>客户端socket数：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="clientSocketLE">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="getCSPB">
      <property name="text">
       <string>获取</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>321</y>
     <width>621</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="sendLabel">
      <property name="text">
       <string>向客户端发送消息（第几个socket，内容）：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="socketNumLE"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="sendLE"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="sendPB">
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
